### DISCUSSION TITLE
Does the newest version no longer have "Duplicate Event"?

### DISCUSSION THREAD

**Discussion by lintonhale** [liedekef/events-made-easy#137] [Q&A]
If not is there some other way I can duplicate an event?  Thanks!

**Comment by liedekef** [comment]
since you marked it as resolved it seems you found the duplicate icon :-)

**Comment by lintonhale** [comment]
Yes, thank you!

On Mon, Apr 3, 2023 at 10:50 AM Franky Van Liedekerke <
***@***.***> wrote:

> since you marked it as resolved it seems you found the duplicate icon :-)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/liedekef/events-made-easy/discussions/137#discussioncomment-5512850>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZIQ4VOB5XLYAIN3EZ4NKLW7MEVZANCNFSM6AAAAAAWRS4QFU>
> .
> You are receiving this because you modified the open/close state.Message
> ID: <liedekef/events-made-easy/repo-discussions/137/comments/5512850@
> github.com>
>


-- 
Linton Hale

Web Developer, Internet Coach, Musician
707-206-5442 ***@***.***

<https://webwatchdawg.com/>
Terms of Service <https://webwatchdawg.com/terms-of-service/> | Privacy
Policy <https://webwatchdawg.com/privacy-policy/>
